pkg/storage/couchdb/couchdbstore.go

```diff
@@ -498,6 +581,35 @@ func (c *CouchDBStore) getRevID(k string) (string, error) {
 	return revIDString, nil
 }
 
+func (c *CouchDBStore) getRevIDs(keys []string) ([]string, error) {
+	rawDocs, err := c.getRawDocs(keys)
```

Comment:
getRawDocs returns the entire contents of the document (as far as I can tell) when you only need the "_rev" field. Can you make a bulk request to simply return the one field?

Reply:
I did some research into CouchDB and Kivik, and while I can't rule it out as being impossible, I haven't found a way of implementing this in a reasonable timeframe for the 0.1.5 release. For now, I've left a TODO and if I've run out of things to do before the release (unlikely) I'll come back to this to see if I can improve performance even more. https://github.com/trustbloc/edge-core/issues/121